src/Oscar.jl

```diff
@@ -470,82 +470,19 @@ include("exports.jl")
 include("printing.jl")
 include("fallbacks.jl")
 
-include("GAP/wrappers.jl")
-
-include("Groups/types.jl")
-include("Groups/perm.jl")
-include("Groups/group_constructors.jl")
-include("Groups/sub.jl")
-include("Groups/homomorphisms.jl")
-include("Groups/cosets.jl")
-include("Groups/GAPGroups.jl")
-include("Groups/pcgroup.jl")
-include("Groups/directproducts.jl")
-include("Groups/matrices/matrices.jl")
-include("Groups/matrices/FiniteFormOrthogonalGroup.jl")
-include("Groups/libraries/libraries.jl")
-include("Groups/gsets.jl")
-include("Groups/MatrixDisplay.jl")
-include("Groups/abelian_aut.jl")
-include("Groups/spinor_norms.jl")
-include("Groups/GrpAb.jl")
-
-include("Rings/integer.jl")
-include("Rings/orderings.jl")
-include("Rings/mpoly.jl")
-include("Rings/mpoly_types.jl")
-include("Rings/mpoly-graded.jl")
-include("Rings/mpoly-ideals.jl")
-include("Rings/groebner.jl")
-include("Rings/solving.jl")
-include("Rings/MPolyQuo.jl")
-include("Rings/mpoly-nested.jl")
-include("Rings/FractionalIdeal.jl")
-
-include("Rings/mpoly-affine-algebras.jl")
-
-include("Rings/special_ideals.jl")
-
-include("Rings/MPolyMap/MPolyAnyMap.jl")
-include("Rings/MPolyMap/MPolyRing.jl")
-include("Rings/MPolyMap/MPolyQuo.jl")
-include("Rings/MPolyMap/AffineAlgebras.jl")
-
-include("Rings/mpoly-local.jl")
-include("Rings/localization_interface.jl")
-include("Rings/mpoly-localizations.jl")
-include("Rings/mpolyquo-localizations.jl")
-include("Rings/MPolyMap/flattenings.jl")
-include("Rings/FinField.jl")
-include("Rings/NumberField.jl")
-include("Rings/FunctionField.jl")
-include("Rings/AbelianClosure.jl")
-
-include("Rings/PBWAlgebra.jl")
-include("Rings/PBWAlgebraQuo.jl")
-include("Rings/FreeAssAlgIdeal.jl")
-
-
-include("GAP/customize.jl")
-include("GAP/gap_to_oscar.jl")
-include("GAP/oscar_to_gap.jl")
-include("GAP/iso_gap_oscar.jl")
-include("GAP/iso_oscar_gap.jl")
+
+include("Groups/Groups.jl")
+
+include("Rings/Rings.jl")
+include("GAP/GAP.jl")
```

Comment:
Many files in `Groups/` depend on the `GAPWrap` module in `GAP/wrappers.jl`. Could you try to reorder the includes accordingly?

Reply:
It seems fine though?